pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -459,13 +467,14 @@ def _set_deepspeed_activation_checkpointing(self):
 
     def _initialize_deepspeed_inference(self, model):
```

Comment:
@SeanNaren does this method not need to set the trainer `optimizer`/`lr_schedulers` attributes at the end? just like we do for `_initialize_deepspeed_train`